js/index.js

```diff
@@ -675,20 +675,28 @@ programButton.onclick = async () => {
 }
 
 async function downloadAndFlash(fileURL) {
-    let data = await utilities.getImageData(fileURL);
+	let data = await utilities.getImageData(fileURL);
+	let strOffset = config[deviceTypeSelect.value].offset;
+	let offset = 0x0000;
+	if ( strOffset !== undefined )
+	{
+	    offset = parseInt(strOffset);
+	}
```

Comment:
Could we simplify this logic using the nullish coalescing operator?
```suggestion
	let offset = parseInt(config[deviceTypeSelect.value].offset ?? "0x0000");
```